release/downloadIstioCandidate.sh

```diff
@@ -78,25 +75,23 @@ NAME="istio-$ISTIO_VERSION"
 URL="https://github.com/istio/istio/releases/download/${ISTIO_VERSION}/istio-${ISTIO_VERSION}-${OSEXT}.tar.gz"
 ARCH_URL="https://github.com/istio/istio/releases/download/${ISTIO_VERSION}/istio-${ISTIO_VERSION}-${OSEXT}-${ISTIO_ARCH}.tar.gz"
 
-printf "Downloading %s from %s ..." "$NAME" "$URL"
-if ! curl -fsLO "$URL"
-then
-  printf "Failed.\n\nTrying with TARGET_ARCH. Downloading %s from %s ...\n" "$NAME" "$ARCH_URL"
-  if ! curl -fsLO "$ARCH_URL"
-  then
-    printf "\n\n"
-    printf "Unable to download Istio %s at this moment!\n" "$ISTIO_VERSION"
-    printf "Please verify the version you are trying to download.\n\n"
-    exit
-  else
-    filename="istio-${ISTIO_VERSION}-${OSEXT}-${ISTIO_ARCH}.tar.gz"
-    tar -xzf "${filename}"
-    rm "${filename}"
-  fi
-else
+if [ "${OS}" = "Linux" ] ; then
```

Comment:
This change will break 1.5 and below. As of Friday, 1.5 is no longer supported, but I'm not sure if we want to remove user's ability to download it yet. Perhaps we can just reverse the cases below (try with architecture first and without after)? 1.6 and 1.7 both support architectures.

Reply:
Also, these changes should be mirrored for istioctl.

Reply:
> This change will break 1.5 and below.

Ahh, I found this https://github.com/istio/istio.io/pull/7865.

> Perhaps we can just reverse the cases below (try with architecture first and without after)?

Okey, I will update and test it. Thanks.

Reply:
No problem. Thank you. A bit of backstory on that. The istio and istioctl urls are both at istio.io/... These are the same through all releases. Before the change you linked, it would redirect to a file in the current release branch (so 1.5 would look at the file in the 1.7 branch). Now it just redirects to master.

Reply:
Added support for `1.5` and older downloads.